src/apollo/client.js

```diff
@@ -1,50 +1,130 @@
-import { ApolloClient } from 'apollo-client';
-import { InMemoryCache } from 'apollo-cache-inmemory';
+import { ApolloClient, InMemoryCache } from '@apollo/client';
 import { HttpLink } from 'apollo-link-http';
 
 export const client = new ApolloClient({
   link: new HttpLink({
     uri: 'https://polygon.furadao.org/subgraphs/name/quickswap',
   }),
   cache: new InMemoryCache(),
-  shouldBatch: true,
+  queryDeduplication: true,
+  defaultOptions: {
+    watchQuery: {
+      fetchPolicy: 'cache-first',
+    },
+    query: {
+      fetchPolicy: 'cache-first',
```

Comment:
Why fetchPolicy is cache-first? This will pick data from the cache and we will not get the updated information. Same applies on all client instantiations

Reply:
It should be no-cache